pkg/placement/controller.go

```diff
@@ -42,6 +42,8 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```

Comment:
FYI: Since controller-runtime and zap were developed, Kubernetes got klog and logr-style logging (i.e., both leveled and structured). I think modern Kubernetes code uses the structured logging in klog.

Reply:
Are you ok in leaving switching to klog in a further PR just focused on logging infrastructure. ?

Reply:
yes